frontend/src/store/modules/friends.ts

```diff
@@ -0,0 +1,39 @@
+import services from "@/api/services";
+import { GetFriendsRequest, GetFriendsResponse } from "@/protobuf/friend_pb";
+import { RootState } from "@/store";
+import { Module } from "vuex";
+
+export interface FriendsState {
```

Comment:
Not sure if we should rename this module to User? Then in the user store, we can keep store:
- friends
- signed in user (upon login, we set the store with the signed in user)
- user's current geolocation

Thoughts?

Reply:
makes sense! will try